### PR TITLE
BUG: Workaround PySide bug.

### LINKED ISSUE #36: Cannot set alignment in GroupBox
Trying to set the `alignment` attribute of a `GroupBox` fails with this traceback:

```
Exception occurred in traits notification handler for object: GroupBox(), trait: qobj, old value: None, new value: <PySide.QtGui.QGroupBox object at 0x117474998>
Traceback (most recent call last):
  File "build/bdist.macosx-10.6-x86_64/egg/traits/trait_notifiers.py", line 340, in __call__
  File "/Users/pberkes/egit/qt_binder/qt_binder/binder.py", line 611, in _qobj_changed
    self.trait_set(**values)
  File "build/bdist.macosx-10.6-x86_64/egg/traits/has_traits.py", line 1545, in trait_set
  File "/Users/pberkes/egit/qt_binder/qt_binder/binder.py", line 162, in set
    old = self.get(object, name)
  File "/Users/pberkes/egit/qt_binder/qt_binder/binder.py", line 143, in get
    value = self.meta_prop.read(qobj)
RuntimeError: Can't find converter for 'Qt::Alignment'.
```

Example:

```
from traits.api import HasStrictTraits
from traitsui.api import View

from qt_binder.bound_editor import Bound
from qt_binder.qt import QtCore
from qt_binder.raw_widgets import GroupBox, Label, VBoxLayout


class Model(HasStrictTraits):

    def default_traits_view(self):
        traits_view = View(
            Bound(
                GroupBox(
                    VBoxLayout(
                        Label(text='Hi'),
                    ),
                    title=u'Time',
                    alignment=QtCore.Qt.AlignRight,
                ),
            ),
            resizable=True,
            width=300,
            height=200,
        )
        return traits_view


if __name__ == '__main__':
    m = Model()
    m.configure_traits()
```

### PR DESCRIPTION
**Pull request by rkern**
Fixes #36

@pberkes

### CODE REVIEW

File: qt_binder/tests/test_raw_widgets.py [hunk 1]
```diff
@@ -70,3 +63,20 @@ def test_lookup_widget(self):
...
+        box.construct()
+        box.configure()
+        self.assertIsInstance(box.qobj, QtGui.QGroupBox)
+
+    def test_groupbox_alignment_works(self):
+        # PySide has a bug. Ensure that we work around it.
+        box = GroupBox()
+        box.construct()
+        box.configure()
+        _ = box.alignment
```
Comment (pberkes) [inline qt_binder/tests/test_raw_widgets.py]: Can be removed as far as I can tell.

**Comment by pberkes** [comment]
flake8 error

File: qt_binder/tests/test_raw_widgets.py (hunk 1 above)
Comment (rkern) [inline qt_binder/tests/test_raw_widgets.py]: The error actually occurs during the `.get()`. `.set()` does happen to call `.get()` in order to get the `old` value to pass to the trait change handler, but there is no guarantee that it must go through the same code path; that's just the current implementation detail. So I test both independently.

**Comment by rkern** [comment]
Fixed.

**Comment by pberkes** [comment]
LGTM

**Pull request merged**